src/puck.ts

```diff
@@ -271,11 +287,23 @@ export class RikaiPuck {
     target.dispatchEvent(mouseEvent);
   };
 
+  private clickState: ClickState = { kind: 'idle' };
+  private static readonly doubleClickHysteresis = 300;
```

Comment:
TIL!

Reply:
I've only heard of it in the context of how MRI machines work, but it's mentioned 6 times in the [UIEvents W3C spec](https://w3c.github.io/uievents/) and specifically for the timing of double-clicks, so it seemed like the term to use!